app/screens/select_server/select_server.js

```diff
@@ -439,6 +442,10 @@ export default class SelectServer extends PureComponent {
                                 />
                             </View>
                             <TextInput
+                                accessibilityLabel={formatMessage({
+                                    id: 'mobile.components.select_server_view.enterServerUrl',
+                                    defaultMessage: 'Enter Server URL',
+                                })}
```

Comment:
This additional label is unnecessary, since tapping the `TextInput` automatically would read aloud the placeholder text, or whatever text is already in the field. Also, reading aloud 'Enter Server URL' isn't relevant to the `TextInput`, but to the preceding `FormattedText` element, which is already covered by accessibility and will already read aloud "Enter Server URL" when touched.

Reply:
Relevant fields are free to be touched now. Nice contribution @RyanCommits

Reply:
Agreed... Fixed in new commit, thanks.